skimage/transform/_warps.py

```diff
@@ -18,14 +17,27 @@
 )
 
 
+def _multichannel_default(multichannel, ndim):
+    if multichannel is not None:
+        return multichannel
+    else:
+        warn('The default multichannel argument (None) is deprecated. '
+             'Specifiy True or False explicitly.')
+        # utility for maintaining previous color image default behavior
+        if ndim == 3:
+            return True
+        else:
+            return False
+
+
 def resize(image, output_shape, order=1, mode=None, cval=0, clip=True,
            preserve_range=False):
```

Comment:
@soupault yes I'm happy to discuss it. Notice that I consider this a different discussion to what we've had before: when the argument can be inferred unambiguously, vs when we are just guessing.

At any rate, I'm happy to have the discussion, but I don't think it should hold this PR up. Perhaps you can raise an issue afterwards? And it can cover existing cases of guessing.